<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advancedsearch</class>
 <widget class="QWidget" name="Advancedsearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>高级搜索</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>54</height>
    </rect>
   </property>
   <property name="title">
    <string>卫星</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="duplicatesEnabled">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="multiSatcheckBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>多颗卫星</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>时间</string>
   </property>
   <widget class="QLineEdit" name="startdateEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2013-01-01 00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startdateButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(70,130,180);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="QFramer.qrc">
      <normaloff>:/skin/icons/light/appbar.calendar.png</normaloff>:/skin/icons/light/appbar.calendar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="enddateEdit">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2013-01-31 00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enddateButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(70,130,180);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="QFramer.qrc">
      <normaloff>:/skin/icons/light/appbar.calendar.png</normaloff>:/skin/icons/light/appbar.calendar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>23</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>起始时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>21</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结束时间</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加条件</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>287</width>
     <height>691</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>461</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>选择参数</string>
   </property>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>281</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="selectlistWidget_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>191</width>
      <height>281</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="delButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QFramer.qrc"/>
 </resources>
 <connections/>
</ui>
